app/services/rbac_api.rb

```diff
@@ -1,32 +1,39 @@
 # frozen_string_literal: true
 
+require 'insights-rbac-api-client'
+
 # This class is meant to handle calls to the RBAC light API
 class RbacApi
   def initialize(b64_identity)
-    @url = URI.parse("#{URI.parse(Settings.rbac_url)}"\
-                     "#{Settings.path_prefix}/rbac/v1/access/")
+    RBACApiClient.configure do |config|
+      config.host = Settings.rbac_url
+      config.username = Settings.rbac_username
+      config.password = Settings.rbac_password
+    end
+    @api_instance = RBACApiClient::AccessApi.new
```

Comment:
Looks more conventional:
```suggestion
    @client = RBACApiClient::AccessApi.new
```